index.bs

```diff
@@ -1725,10 +1735,13 @@ partial interface MLGraphBuilder {
         1. If |input| is a 4-D tensor of the *"nchw"* layout, set |options|.axis to 1.
         1. If |input| is a 4-D tensor of the *"nhwc"* layout, set |options|.axis to 3.
     1. Let |result| be an {{MLOperand}} representing the results. It may use the same underlying data as |input|.
-    1. Issue a request to the underlying platform to initialize the batch normalization, passing the arguments |input|, |mean|, |variance| and |options| and given |result| to store the results and |options|. Wait for completion.
-        <div class="informalsteps">
-            1. If |options|.activation [=map/exists=], implementations MAY use it to optimize the operation flow.
-        </div>
+    1. If any of the following sub-steps fail, throw an "{{OperationError}}" {{DOMException}} and stop.
+        1. Make a request to the underlying platform to initialize the batch normalization:
+            1. Create an [=implementation-defined=] platform operator |batchNormImpl| for this method, given |input|, |mean|, |variance| and |options|.
+            1. Register |result| as output to |batchNormImpl|.
+                <div class="informalsteps">
+                    1. If |options|.activation [=map/exists=], implementations MAY use it to optimize the operation flow.
```

Comment:
if activation exists, we should register it to platform operator batchNormImpl

Reply:
OK, I was pondering that, too.